packages/gafl-webapp-service/src/pages/concessions/disability/disability-concession.njk

```diff
@@ -1,4 +1,5 @@
 {% extends "standard-form.njk" %}
+
```

Comment:
Doing this in quite a few pages, reset these to the state they were in to reduce the number of files in this PR